_pages/oidc.md

```diff
@@ -36,17 +36,18 @@ sidenav:
 
 ## Getting started
 
-### Choosing an authentication method
+### Client authentication
 
-Login.gov supports two ways of authenticating clients: **private_key_jwt** and **PKCE**.
+Login.gov supports only **private_key_jwt** for client authentication.
```

Comment:
if PKCE is not considered client authentication, what is the name of the umbrella category where we can offer both private_key_jwt and PKCE as options to integrate with us?

Reply:
I tried to keep this PR simple and to describe the current behavior accurately. PKCE is also recommended even if you are using any form of client authentication including private_key_jwt, but that's a different discussion to be had 😄  This PR doesn't attempt to change any behavior of the current system, just avoids referring to PKCE as a form of client authentication. If you wanted to have two options side by side, then I would say the forms of client authentication are **private_key_jwt** and **none**, where PKCE is required if you are not including client authentication.

Reply:
Ah, I see. We'll probably go with that private_key_jwt/none approach then, and keep the bulleted list

Reply:
Alright, I updated the PR to bring back the bullet list.

Reply:
Ah thanks for letting us know, we'll fix the link on our security page, but in the meantime here is the right HackerOne project: https://hackerone.com/gsa_bbp?type=team